ctsm_filenames.csv

```diff
@@ -1,12 +1,12 @@
 ﻿,dir_name,old_fname,new_fname,old_string,new_string
 0,/glade/scratch/negins/ctsm_0110/src/main,column_varcon.F90,ctsm_ColumnVarCon.F90,column_varcon,ctsm_ColumnVarCon
 1,/glade/scratch/negins/ctsm_0110/src/main,initGridCellsMod.F90,ctsm_InitGridCells.F90,initGridCellsMod,ctsm_InitGridCells
-2,/glade/scratch/negins/ctsm_0110/src/main,clm_varsur.F90,ctsm_VarSur.F90,clm_varsur,ctsm_VarSur
+2,/glade/scratch/negins/ctsm_0110/src/main,clm_varsur.F90,ctsm_VarSur.F90,clm_instur,ctsm_VarSur

```

Comment:
Not sure why this module name disagreed with the file name, but it did.

Reply:
What do you think about making the file names and module names consistent?

Reply:
Yes, we should make them consistent moving forward. My point in pointing this out is that your search & replaces won't work unless you have the correct old names.

Reply:
Thanks for pointing this out but the new name will be similar to filenames: 
i.e. module name will be changed from `clm_instur` to ctsm_VarSur`
Is my understanding correct?

Reply:
Yes